[2024-04-20T17:14:34.974+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-20T17:14:30.743890+00:00 [queued]>
[2024-04-20T17:14:34.992+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-20T17:14:30.743890+00:00 [queued]>
[2024-04-20T17:14:34.992+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2024-04-20T17:14:35.007+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): s3_upload> on 2024-04-20 17:14:30.743890+00:00
[2024-04-20T17:14:35.015+0000] {standard_task_runner.py:57} INFO - Started process 112 to run task
[2024-04-20T17:14:35.019+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_reddit_pipeline', 's3_upload', 'manual__2024-04-20T17:14:30.743890+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/reddit_dag.py', '--cfg-path', '/tmp/tmpz381j_cc']
[2024-04-20T17:14:35.022+0000] {standard_task_runner.py:85} INFO - Job 24: Subtask s3_upload
[2024-04-20T17:14:35.066+0000] {task_command.py:416} INFO - Running <TaskInstance: etl_reddit_pipeline.s3_upload manual__2024-04-20T17:14:30.743890+00:00 [running]> on host a5ce8015e481
[2024-04-20T17:14:35.166+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Tawfik Yasser' AIRFLOW_CTX_DAG_ID='etl_reddit_pipeline' AIRFLOW_CTX_TASK_ID='s3_upload' AIRFLOW_CTX_EXECUTION_DATE='2024-04-20T17:14:30.743890+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-20T17:14:30.743890+00:00'
[2024-04-20T17:14:35.906+0000] {logging_mixin.py:151} INFO - Bucket already exists
[2024-04-20T17:14:36.376+0000] {logging_mixin.py:151} INFO - File uploaded
[2024-04-20T17:14:36.377+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-04-20T17:14:36.388+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_reddit_pipeline, task_id=s3_upload, execution_date=20240420T171430, start_date=20240420T171434, end_date=20240420T171436
[2024-04-20T17:14:36.436+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-04-20T17:14:36.451+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
